packages/cmf/src/sagas/http.js

```diff
@@ -111,6 +117,74 @@ export function httpFetch(url, config, method, payload) {
 		.then(handleHttpResponse)
 		.catch(handleError);
 }
+export function cloneError(initialError, resultError) {
+	const errorToEnrich = Object.assign({}, resultError);
+	return initialError.stack.response
+		.clone()
+		.text()
+		.then(response => {
+			try {
+				errorToEnrich.stack.response = response;
+				errorToEnrich.stack.messageObject = JSON.parse(response);
+				return Promise.resolve(errorToEnrich);
+			} catch (exception) {
+				errorToEnrich.stack.messageObject = 'Parsing response error';
+				return Promise.reject(errorToEnrich);
+			}
+		});
+}
+export function* errorProcessing(error, httpAction) {
+	const errorObject = {
+		name: error.name,
+		message: error.description || error.message,
+		number: error.number,
+		stack: error.stack,
+	};
+	const clone = get(error, 'stack.response.clone');
+	if (!clone) {
+		yield put(onJSError(errorObject, httpAction));
+	} else {
+		// clone the response object else the next call to text or json
+		// triggers an exception Already use
+		const clonedError = cloneError(error);
+		if (httpAction.onError) {
+			yield put(onActionError(httpAction, clonedError));
+		}
+
+		if (typeof httpAction.onError !== 'function') {
+			yield put(onHttpError(httpAction, clonedError));
+		}
+	}
+}
+
+export const handleDefaultConfiguration = curry((defaultConfig, config) =>
+	mergeCSRFToken(defaultConfig)(config),
+);
+
+function normalizeParameters(
+	urlOrBag,
+	config,
+	method = HTTP_METHODS.GET,
+	payload,
+	cmf,
+	defaultConfig,
+) {
+	const configEnhancer = handleDefaultConfiguration(defaultConfig);
+	if (isObject(urlOrBag)) {
+		return {
+			...urlOrBag,
+			config: configEnhancer(urlOrBag.config),
+		};
+	}
+
+	return {
```

Comment:
We should ensure url is a string, either throw an new Error

Reply:
I prefer to let developer use the signature they need, and avoid error but a console.warn about a migration would be fine to me.